### ISSUE TITLE
how to reducer the params for fc layer in decoder?

### ISSUE THREAD

**Issue by zimenglan-sysu-512** [tancik/StegaStamp#25]
hi @tancik 
since the `fc layer` in `decoder` is too larger (13*13 * 128 * 500 = 10,816,000), so i want use a conv(dim=1024) to increase the dimension and then use `tf.reduce_mean(x, axis=[1, 2])`, but in the first 1w iterations, the `bit_acc` always gets around 0.5, and the `str_acc` gets zero. so how to reduce the params for the `fc layer`, at the same time, it can get good results?

**Comment by ggohem** [comment]
> hi @tancik
> since the `fc layer` in `decoder` is too larger (13*13 * 128 * 500 = 10,816,000), so i want use a conv(dim=1024) to increase the dimension and then use `tf.reduce_mean(x, axis=[1, 2])`, but in the first 1w iterations, the `bit_acc` always gets around 0.5, and the `str_acc` gets zero. so how to reduce the params for the `fc layer`, at the same time, it can get good results?

Hey, is this problem solved?  @zimenglan-sysu-512
